### PR TITLE
solution - AK

### PR DESCRIPTION
**Pull request by AnnaKabatova**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,5 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_DB=POSTGRES_DB
+POSTGRES_USER=POSTGRES_USER
+POSTGRES_PASSWORD=POSTGRES_PASSWORD
+SECRET_KEY=SECRET_KEY
```
Comment (ArsenPidhoretskyi) [inline .env.sample:5]: Add emply line

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    django-user
+
+RUN chown -R django-user:django-user /vol/
+RUN chmod -R 755 /vol/web/
+
+USER django-user
```
Comment (ArsenPidhoretskyi) [inline Dockerfile:27]: Empty line

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      - .env
+    depends_on:
+      - db
+
+  db:
+    image: postgres:14-alpine
+    ports:
+      - "5433:5432"
+    env_file:
+      - .env
```
Comment (ArsenPidhoretskyi) [inline docker-compose.yml:25]: Empty line

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
don't work on the master branch - bad practice
https://github.com/mate-academy/py-task-guideline/blob/main/README.md#:~:text=git%20checkout%20%2Db%20develop

**Pull request closed without merging**